### ISSUE TITLE
max

### ISSUE THREAD

**Issue by srcorsi-USGS** [USGS-R/toxEval#257]
double check: change "max" to "maximum" throughout unless it refers to something that contextually needs to be "max" (the max function or if an argument is assigned max)

**Comment by ldecicco-USGS** [comment]
I'm looking through the help files and vignettes. Do you also want the labels on the graphs to be "maximum" or "max" (I'd prefer "max")

**Comment by ldecicco-USGS** [comment]
Open a separate issue if you'd like the labels to read "maximum"...I'm going to close this one once my PR is merged because I went through and got the help files/vignettes.
